src/game/AI/ScriptDevAI/scripts/outland/hellfire_citadel/hellfire_ramparts/boss_watchkeeper_gargolmar.cpp

```diff
@@ -136,6 +136,15 @@ struct boss_watchkeeper_gargolmarAI : public CombatAI
                 if (m_creature->GetHealthPercent() < 40.0f)
                 {
                     DoScriptText(SAY_HEAL, m_creature);
+
+                    CreatureList watcherList;
+                    GetCreatureListWithEntryInGrid(watcherList, m_creature, NPC_HELLFIRE_WATCHER, 100.0f);
+                    for (Creature* watcher: watcherList)
```

Comment:
Now you are missing a space :D But in spirit of cooperation I will fix that myself.